gen/internals.py

```diff
@@ -284,6 +309,13 @@ class SkipError(Exception):
     pass
 
 
+# has a deterministic chunk before and after the variable name so we can deterministically get out
+# the name
+LATE_BIND_PLACEHOLDER_START = 'LATE_BIND_PLACEHOLDER_START_'
```

Comment:
Sounds ok to me. I don't have a strong stance.

I was getting biased from some templating stuff calling `{{` symbols  as delims rather than placeholders.  
http://jinja.pocoo.org/docs/2.9/templates/

> There are a few kinds of delimiters. The default Jinja delimiters are configured as follows: